share/commands/agent-inventory

```diff
@@ -24,5 +24,16 @@
 # @man *-R*: run the agent in completely unparsed mode, with no return code of 1 in case of error. A little faster.
 # @man +
 # @man *-m*: run the agent with multiline output
+# @man +
+# @man *-f*: run the agent even if it is disabled
+
+OPTS=""
+argcount=$#
+# Process all options in order (getopts doesn't set OPTIND in case of error)
+for OPTIND in `seq 1 ${argcount}`
+do
+  getopts ":f" opt # ignore getopt errors
```

Comment:
Shouldn't this just be getopts "f"? I thought the ":" meant "the previous option takes an argument"